lib/main.dart

```diff
@@ -27,9 +27,9 @@ import 'package:ten_ten_one/wallet/withdraw.dart';
 
 import 'package:ten_ten_one/ffi.io.dart' if (dart.library.html) 'ffi.web.dart';
 
-LightningBalance lightningBalanceModel = LightningBalance();
-BitcoinBalance bitcoinBalanceModel = BitcoinBalance();
-SeedBackupModel seedBackupModel = SeedBackupModel();
+LightningBalance lightningBalance = LightningBalance();
```

Comment:
❓ should we add the suffix ChangeNotifier then? similar to the `CfdTradingChangeNotifier` and `WalletChangeNotifier`?

Reply:
I think we should, and we could actually move them somewhere else - those are (part of) the controller for me, not the model. 

This requires a bit more cleanup because what is currently `balance_model` is actually controllers. I will see to address this in a follow up!